lib/HelloSignResource.js

```diff
@@ -26,7 +26,6 @@
 
 var http = require('http');
 var https = require('https');
-var request = require('request');
```

Comment:
Wasn't this already removed? I think your local copy of `v3` might be outdated. Can you merge origin `v3` into your local?

Reply:
That was resolved for future pull requests. I needed to merge prior to these changes.